### ISSUE TITLE
Open xdg-user-dirs are manila

### ISSUE THREAD

**Issue by quequotion** [germanfr/elementary-blue-icons#1]
When a folder is shown open (ie, in files' list view), a manila icon is used for that.

**Comment by germanfr** [comment]
Hi, thanks for reporting. Can you post a screenshot or tell me the icon name that is missing? Thanks.

**Comment by quequotion** [comment]
* [Screenshot](https://i.postimg.cc/mgnBWwL0/Screenshot-from-2020-01-08-06-19-58.png).

* It must be `folder-open(-symbolic)`:
```
elementary-icon-theme /usr/share/icons/elementary/places/128/folder-open.svg
elementary-icon-theme /usr/share/icons/elementary/places/16/folder-open.svg
elementary-icon-theme /usr/share/icons/elementary/places/24/folder-open.svg
elementary-icon-theme /usr/share/icons/elementary/places/32/folder-open.svg
elementary-icon-theme /usr/share/icons/elementary/places/48/folder-open.svg
elementary-icon-theme /usr/share/icons/elementary/places/64/folder-open.svg
elementary-icon-theme /usr/share/icons/elementary/status/symbolic/folder-open-symbolic.svg
```

**Comment by quequotion** [comment]
Seems like only xdg-user-dirs (or bookmarked folders?) are manilla when open; this needs a more specific title.

Not all of them have a symbolic tag: `Desktop/` (no tag), `Documents/`, `Downloads/`, `Music/`, `Pictures/`, `Public/`, and `Videos/` are manilla when open, ordinary folders are blue.

See `ROMS/` in [screenshot](https://i.postimg.cc/mgnBWwL0/Screenshot-from-2020-01-08-06-19-58.png).

**Comment by germanfr** [comment]
Thank you, I'll check those out :+1:

**Comment by germanfr** [comment]
Sorry for the delay. I added (I hope all) the missing icon files. You should have an update now from the AUR waiting for you. Feel free to reopen this issue if you find any of these icons is still missing.

**Comment by quequotion** [comment]
Thank you!

Unfortunately `Desktop/` is still manilla when open; the rest are taken care of.

Github won't let me reopen the issue :(

**Comment by germanfr** [comment]
Hi. This theme didn't have any special icon for the desktop. I have also not found any open variation of the desktop icon. When I drag a file to the desktop folder using Nautilus or Nemo, the icon does not change. I guess you are using the elementary file manager. If only you could tell me the name of the icon your file manager is using for it and it is missing, I could add it as a symbolic link to the `folder-open` icon. Greetings.

**Comment by quequotion** [comment]
Yes, I am using elementary's Files.

I think it's "user-desktop-open". It doesn't have any tag, so a symlink should be fine!

I note you already have "user-desktop" (closed folder) in blue; could save 12.5kb by symlinking it to "folder" instead--same with "inode-directory" (there doesn't seem to be any open version, even in elementary's theme); neither icon has a tag, so they are the same as ordinary folder icons.

**Comment by quequotion** [comment]
All of these folders are blue both open and closed; probably been that way for a while!

Thank you!
